app/scenes/channel/channel_dropdown/channel_dropdown.js

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {PropTypes} from 'react';
+import {ScrollView, Text, TouchableOpacity, View} from 'react-native';
+import Icon from 'react-native-vector-icons/FontAwesome';
+
+function ChannelDropdown(props) {
+    const items = [{
+        title: 'View Info',
+        action: () => true
+    }, {
+        title: 'Notification Preferences',
+        action: () => true
+    }, {
+        title: 'Add Members',
+        action: () => true
+    }, {
+        title: 'View Members',
+        action: () => true
+    }, {
+        title: 'Edit Channel Header',
+        action: () => true
+    }, {
+        title: 'Edit Channel Purpose',
+        action: () => true
+    }, {
+        title: 'Rename Channel',
+        action: () => true
+    }, {
+        title: 'Leave Channel',
+        action: () => true
+    }, {
+        title: 'Add to Favorites',
+        action: () => true
+    }];
+
+    return (
+        <View style={{flex: 1}}>
+            <ScrollView>
+                <View style={{flex: 1, alignItems: 'center'}}>
+                    {items.map((item, index) => (
+                        <TouchableOpacity
+                            key={index}
+                            style={{alignSelf: 'stretch', height: 50, justifyContent: 'center', borderBottomWidth: 1, borderBottomColor: '#fff', marginHorizontal: 20}}
```

Comment:
in this case the style does not have to compute anything, maybe creating a `StyleSheet` either at the beginning or at the on the file to set this style would be better